docs/dev_guide/diagnostic_ref/dpct1047.rst

```diff
@@ -25,3 +25,51 @@ Suggestions to Fix
 
 If the matrix P is only used in library API, ignore this warning. If P is used
 other ways, you may need to adjust the value of P.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void foo(cublasHandle_t handle, float **a_array, float **b_array, int *p_array,
+             int *info_array) {
+      cublasSgetrfBatched(handle, 2, a_array, 2, p_array, info_array, 1);
+      cublasSgetrsBatched(handle, CUBLAS_OP_N, 2, 2, a_array, 2, p_array, b_array,
+                          2, info_array, 1);
+    }
+
+results in the following migrated SYCL code:

```

Comment:
Add "*" to SYCL - not yet attributed on page. Need to attribute first use only.

Reply:
done

Reply:
done